ibm_mq/datadog_checks/ibm_mq/queue_metric_collector.py

```diff
@@ -0,0 +1,181 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+from six import iteritems
+
+from datadog_checks.base import AgentCheck, ensure_bytes, ensure_unicode
+from datadog_checks.ibm_mq.metrics import GAUGE
+
+from . import metrics
+
+try:
+    import pymqi
+except ImportError as e:
+    pymqiException = e
+    pymqi = None
+else:
+    # Since pymqi is not be available/installed on win/macOS when running e2e,
+    # we load the following constants only pymqi import succeed
+    SUPPORTED_QUEUE_TYPES = [pymqi.CMQC.MQQT_LOCAL, pymqi.CMQC.MQQT_MODEL]
+
+
+class QueueMetricCollector:
```

Comment:
```suggestion
class QueueMetricCollector(object):
```
Since we are still supporting Python 2